src/Nethermind/Nethermind.Init/Steps/EthereumStepsManager.cs

```diff
@@ -201,6 +177,75 @@ private void RunOneRoundOfInitialization(CancellationToken cancellationToken)
             return step;
         }
 
+        private Task RunPluginBeforeStep(IStep step)
+        {
+            Stopwatch stopwatch = Stopwatch.StartNew();
```

Comment:
would extract this stopwatch from here, the idea of stopwatch was to check how long the whole step takes. Now we measure before/after seperately